src/lbaf/IO/configurationValidator.py

```diff
@@ -27,61 +33,99 @@ def __init__(self, config_to_validate: dict, logger: Logger):
         self.__config_to_validate = config_to_validate
         self.__skeleton = Schema({
             Or("from_data", "from_samplers", only_one=True): dict,
-            "work_model": {"name": And(str, lambda c: c in ALLOWED_WORK_MODELS,
-                                       error=f"{get_error_message(ALLOWED_WORK_MODELS)} needs to be chosen"),
-                           "parameters": {"alpha": float,
-                                          "beta": float,
-                                          "gamma": float}},
+            "work_model": {
+                "name": And(
+                    str,
+                    lambda c: c in ALLOWED_WORK_MODELS,
+                    error=f"{get_error_message(ALLOWED_WORK_MODELS)} must be chosen"),
+                "parameters": {
+                    "alpha": float,
+                    "beta": float,
+                    "gamma": float}},
             "algorithm": {
-                "name": And(str, lambda d: d in ALLOWED_ALGORITHMS,
-                            error=f"{get_error_message(ALLOWED_ALGORITHMS)} needs to be chosen"),
-                Optional("parameters"): dict
-            },
+                "name": And(
+                    str,
+                    lambda d: d in ALLOWED_ALGORITHMS,
+                    error=f"{get_error_message(ALLOWED_ALGORITHMS)} must be chosen"),
+                Optional("parameters"): dict},
             "output_file_stem": str,
-            "n_ranks": And(int, lambda x: x > 0, error="Should be type of 'int' and > 0"),
-            Optional("exodus"): {"x_procs": And(int, lambda x: x > 0, error="Should be type of 'int' and > 0"),
-                                 "y_procs": And(int, lambda x: x > 0, error="Should be type of 'int' and > 0"),
-                                 "z_procs": And(int, lambda x: x > 0, error="Should be type of 'int' and > 0")},
+            "n_ranks": And(
+                int,
+                lambda x: x > 0,
+                error="Should be of type 'int' and > 0"),
+            Optional("generate_meshes"): {
+                "x_ranks": And(int, lambda x: x > 0,
+                               error="Should be of type 'int' and > 0"),
+                "y_ranks": And(int, lambda x: x > 0,
+                               error="Should be of type 'int' and > 0"),
+                "z_ranks": And(int, lambda x: x > 0,
+                               error="Should be of type 'int' and > 0"),
+                "object_jitter": And(float, lambda x: abs(x) < 1.0,
```

Comment:
If this is not optional it should be added to unit tests and acceptance test and it's configuration as well

Reply:
It is optional.